tmhOAuth.php

```diff
@@ -45,6 +45,9 @@ public function reconfigure($config=array()) {
         'token'                      => '',
         'secret'                     => '',
 
+        // RSA private key (for RSA-SHA1 and RSA-SHA256 methods)
+        'private_key_pem'            => '',
```

Comment:
openssl_sign is a messy function as it overloads the types that can be used for the key and algorithm variables.
it's possible to send a resource instead of a string as the private_key_pem but the implementation you are using assumes it's string. wanna call that out in the comment - that if used this is expecting a string representation of a private key and not a file resource?